src/app/pages/courses/courses.page.html

```diff
@@ -1,14 +1,18 @@
-<app-page-container>
-  <app-breadcrumbs></app-breadcrumbs>
-  <div class="search-container">
-    <input type="text" id="courseSearch" />
-    <button>Search</button>
-  </div>
-  <div class="add-course-container">
-    <button><fa-icon [icon]="addCourseIcon"></fa-icon>Add course</button>
-  </div>
-  <app-courses-container [courses]="courses"></app-courses-container>
-  <div class="load-more">
-    <button class="load-more-button">Load more</button>
-  </div>
-</app-page-container>
+<div class="search-container">
```

Comment:
definately should be a separate component, search features usually have some logic etc like debouncing , it will pollute courses.page component